include/grpcpp/impl/codegen/async_unary_call.h

```diff
@@ -66,26 +72,133 @@ class ClientAsyncResponseReaderInterface {
 };
 
 namespace internal {
-template <class R>
-class ClientAsyncResponseReaderFactory {
+
+class ClientAsyncResponseReaderHelper {
  public:
   /// Start a call and write the request out if \a start is set.
   /// \a tag will be notified on \a cq when the call has been started (i.e.
   /// intitial metadata sent) and \a request has been written out.
   /// If \a start is not set, the actual call must be initiated by StartCall
   /// Note that \a context will be used to fill in custom initial metadata
-  /// used to send to the server when starting the call.
+  /// used to send to the server when starting the call. Optionally pass in a
+  /// base class for request and response types so that the internal functions
+  /// and structs can be templated based on that, allowing reuse across RPCs
+  /// (e.g., MessageLite for protobuf). Since constructors can't have an
+  /// explicit template parameter, the last argument is a bogus to the
```

Comment:
s/bogus/bonus/

Reply:
I actuallly meant "bogus" but didn't give a noun after that. I am changing it to "extraneous parameter".